resources/kcp/charts/kyma-environment-broker/templates/environments-cleanup-job.yaml

```diff
@@ -98,7 +98,7 @@ spec:
                 mountPath: /secrets/cloudsql-instance-credentials
                 readOnly: true
           - name: cloudsql-proxy
-            image: gcr.io/cloudsql-docker/gce-proxy:1.14
+            image: gcr.io/cloudsql-docker/gce-proxy:1.19.1
             command:
```

Comment:
for images > 1.14 there is no bash anymore. The current command will fail. 
please replace:
```yaml
command:
  - /bin/sh
args:
  - -c
  - "trap 'exit 0' SIGINT; /cloud_sql_proxy -instances={{ .Values.global.database.managedGCP.instanceConnectionName }}=tcp:5432 -credential_file=/secrets/cloudsql-instance-credentials/credentials.json"
```
to:
```yaml
command: [ "/cloud_sql_proxy",
                     "-instances={{ .Values.global.database.managedGCP.instanceConnectionName }}=tcp:5432",
                     "-credential_file=/secrets/cloudsql-instance-credentials/credentials.json" ]
```

it was used to kill `cloud_sql` container to finish cron job (without this, one of the containers still live and cron job will never end)

Reply:
Ok! Done